runtime/testy/stateManager.py

```diff
@@ -143,7 +143,7 @@ def send_addr(self, process_name):
     self.processMapping[process_name].send(self.state["dawn_addr"][0])
 
   def student_upload(self):
-    #TODO Implement Student Upload Confirmation
+    self.badThingsQueue.put(BadThing(sys.exc_info(), None, BAD_EVENTS.ENTER_IDLE, False))
     self.processMapping[PROCESS_NAMES.TCP_PROCESS].send([ANSIBLE_COMMANDS.STUDENT_UPLOAD, True])
```

Comment:
Theoretically we would send False if we are not ready for an upload, but as far as I can tell as long as we kill student code we should always be ready, so I guess this will just always be True.